app/models/Subscriber.scala

```diff
@@ -36,13 +36,28 @@ case class Subscriber(id: Int,  // DB key
     }
   }
 
-  def hasInterest(schemeId: Int) = {
+  def interestIn(schemeId: Int) = {
     DB.withConnection { implicit c =>
       SQL("select * from interest where subscriber_id = {sub_id} and scheme_id = {scheme_id}")
-      .on('sub_id -> id, 'scheme_id -> schemeId).as(Interest.interest.singleOpt).isDefined
+      .on('sub_id -> id, 'scheme_id -> schemeId).as(Interest.interest.singleOpt)
     }
   }
 
+  def hasInterest(schemeId: Int) = interestIn(schemeId).isDefined
```

Comment:
General Scala question. Is there a way to ask for the opposite of this convenience method or would we need a separate method to handle this (if we had a reason to need/want it): `Subscriber.!hasInterest?`